docs/layouts/_default/header.html

```diff
@@ -0,0 +1,22 @@
+{{ define "main" }}
+<section class="row td-search-result">
+    <div class="col-12 col-md-8 offset-md-2">
+        <h2 class="ml-4">{{ .Title }}</h2>
+        {{ with .Site.Params.gcs_engine_id }}
+        <script>
+            (function() {
+                var cx = '{{ . }}';
+                var gcse = document.createElement('script');
+                gcse.type = 'text/javascript';
+                gcse.async = true;
+                gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
+                var s = document.getElementsByTagName('script')[0];
+                s.parentNode.insertBefore(gcse, s);
+            })();
+        </script>
+        <gcse:search></gcse:search>
+        {{ end }}
+    </div>
+</section>
+
+{{ end }}
```

Comment:
Missing newline at end of file

```suggestion
{{ end }}

```